### PR TITLE
Added module TARGETURI support.

### PR DESCRIPTION
**Pull request by aushack**
Noticed it was missing

### CODE REVIEW

File: modules/auxiliary/admin/http/tomcat_utf8_traversal.rb [hunk 1]
```diff
@@ -64,7 +65,7 @@ def find_files(files)
       res = send_request_raw(
         {
           'method'  => 'GET',
-          'uri'     => try + files,
+          'uri'     => normalize_uri(datastore['TARGETURI']) + try + files,
```
Comment (bcoles) [inline modules/auxiliary/admin/http/tomcat_utf8_traversal.rb]: `normalize_uri` should usually encompass all parts of the path. Although I'm not sure if it's relevant in this instance, due to the traversal sequence.

```suggestion
          'uri'     => normalize_uri(datastore['TARGETURI'], try, files),
```

**Comment by aushack** [comment]
Yeah there is a change it might mess things up, but should be fine with unicode I believe.

**Comment by sempervictus** [comment]
Looks reasonable. Did we ever sort out the interchangeable use of URI and TARGETURI in modules? This might be a good place for that if not.

**Comment by aushack** [comment]
Don't know anymore - easily been a decade since I committed that.

Strangely had a pentest where I needed it, and realised I must have
forgotten TARGETURI (unless it was inherited back then?).

If anyone was so inclined, it is likely there are many other HttpClient
modules absent TARGETURI / normalisation too.

**Review by wvu — APPROVED** [review]

**Comment by wvu-r7** [comment]
# Release Notes

This adds `TARGETURI` support to the `auxiliary/admin/http/tomcat_utf8_traversal` module.

**Pull request merged**